content/uiux/concepts/visual-design-principles/visual-design-principles.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Visual Design Principles'
+Description: 'Visual design principles are guidelines for creating visual designs that are effective, cohesive, and appealing. Common principles include hierarchy, contrast, balance, scale, dominance, unity, and Gestalt.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Visual design principles** are guidelines for creating visual designs that are effective, cohesive, and appealing. Common principles include hierarchy, contrast, balance, scale, dominance, unity, and Gestalt.
+
+- **Hierarchy** relates to the order that a person processes visual information. An interface with a clear hierarchy helps the user prioritize information and systematically view elements on the page. By contrast, a lack of hierarchy creates confusion, as it’s unclear what to focus on.
```

Comment:
Remove blank lines between elements of the bulleted list.

Reply:
Removed